src/mpi/romio/adio/ad_gpfs/bg/ad_bg_pset.c

```diff
@@ -16,14 +16,14 @@
 // #define bridgeringaggtrace 1
 
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
Sorry I could not get the bit wrangling to work with int.  When MPL_bool changes to bool I will definitely use it, I assume at that time the code would be scrubbed anyhow for stdbool.  Could I please keep this for now?  Thanks.